parm/fv3lam_cmaq.xml-org

```diff
@@ -0,0 +1,3953 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@hsinmulin-NOAA This file should be removed from your PR.

Reply:
removed